tools/io-trace-explorer/README

```diff
@@ -0,0 +1,55 @@
+ I/O Trace Explorer
+====================
+
+I/O Trace Explorer is a GUI tool for visualizing I/O traces from WiredTiger, at both the file level
+and at the device level. It is useful for helping us to better understand and optimize our I/O
```

Comment:
I would rephrase this to say: "It helps us better understand and optimize our I/O patterns".